serverless.yml

```diff
@@ -34,7 +34,7 @@ custom:
 functions:
   stac:
     handler: src/lambda.handler
-    timeout: 60
+    timeout: 180
```

Comment:
When stress testing in very large page queries could timeout before GraphQL returned. Limited query size, but keeping the increase for large queries. Specifically certain DAACs have an outsized number of collections compared to other providers.